src/libraries/System.Net.Security/src/System/Net/Security/Pal.Managed/SafeChannelBindingHandle.cs

```diff
@@ -11,7 +11,7 @@ namespace System.Net.Security
     internal sealed class SafeChannelBindingHandle : ChannelBinding
     {
         private const int CertHashMaxSize = 128;
-        private static readonly int s_secChannelBindingSize = Marshal.SizeOf<SecChannelBindings>();
+        private static readonly int s_secChannelBindingSize = sizeof(SecChannelBindings);
```

Comment:
```suggestion
        private static int SecChannelBindingSize => sizeof(SecChannelBindings);
```
No need to cache the constant in a static

Reply:
I had to change it to `Unsafe.SizeOf<SecChannelBindings>()` to avoid the CS0233 "'identifier' does not have a predefined size, therefore sizeof can only be used in an unsafe context" error.

Changed this to property as well.

Reply:
We tend to prefer [wrapping the call in `unsafe` block](https://github.com/dotnet/runtime/pull/78741), but `Unsafe.SizeOf` works too - it is still many times better than Marshal.SizeOf.

Reply:
@jkotas I thought that `unsafe` blocks interfered with some compiler/jit optimizations. I recall seeing PRs which refactored code to remove `unsafe` blocks and it measurably increased perf. But maybe the perf increase was caused by other changes in the PR.

Reply:
Introducing unsafe block alone (with no other code changes) has zero impact on compiler/jit optimizations.

The perf increase that you are remembering must have been caused by some other actual changes in the PR.